pkgs/development/python-modules/langgraph-checkpoint-postgres/default.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  langgraph-checkpoint,
+  psycopg,
+  langgraph-sdk,
+  poetry-core,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "langgraph-checkpoint-postgres";
+  version = "1.0.3";
+  pyproject = true;
+
+  disabled = pythonOlder "3.10";
+
+  src = fetchFromGitHub {
+    owner = "langchain-ai";
+    repo = "langgraph";
+    rev = "refs/tags/checkpointpostgres==${version}";
+    hash = "sha256-U7Bymo+Nj82kwjxN33W2MT10jv+lioZUxIKUt8Yxh/s=";
+  };
+
+  sourceRoot = "${src.name}/libs/checkpoint-postgres";
+
+  build-system = [ poetry-core ];
+
+  dependencies = [
+    langgraph-checkpoint
+    psycopg
+  ];
+
+  pythonImportsCheck = [ "langgraph.checkpoint.postgres" ];
+
+  # Tests require database connection
+  doCheck = false;
```

Comment:
Can we use `postgresqlTestHook` here?
https://github.com/NixOS/nixpkgs/blob/master/doc/hooks/postgresql-test-hook.section.md

Reply:
Fixed.